sphinx/source/docs/user_guide/tools.rst

```diff
@@ -766,7 +761,7 @@ supplied renderer.
 
 It is also often useful to limit the number of elements that can be
 drawn, e.g. when specifying a specific number of regions of interest.
```

Comment:
```suggestion
drawn. For example, when you want to specify certain regions of interest.
```

Reply:
Also see my comment about the BoxEditTool section above. We might want to be a little clearer here by saying:

> For example, when you want to limit the option to use the FreehandDrawTool to certain regions of interest.